docs/faq/index.md

```diff
@@ -28,6 +28,12 @@ If you don't see your question here, please reach out to us on Slack or at
     We're constantly adding new data types and annotation types — if you don't see what you're looking for, reach out
     to us and we'll happily extend our system to support your use case.
 
+??? faq "Does Kolena support file versioning?"
```

Comment:
I'd move this to the "Using Kolena" section below — this topmost "About Kolena" section is intended to answer the high-level questions about what we do.